kml/kml_to_kmb/kml_to_kmb.cpp

```diff
@@ -0,0 +1,47 @@
+#include "kml/serdes.hpp"
+#include "kml/serdes_binary.hpp"
+
+#include "indexer/classificator_loader.hpp"
+
+#include "coding/file_reader.hpp"
+#include "coding/file_writer.hpp"
+
+#include <iostream>
+
+
+int main(int argc, char** argv) {
+  if (argc < 2) {
+    std::cout << "Converts kml file to kmb\n";
+    std::cout << "Usage: " << argv[0] << " path_to_kml_file\n";
+    return 1;
+  }
+  // TODO: Why bookmarks serialization requires classifier?
```

Comment:
Please any actionable tickets (i.e. "Investigate why bookmarks serialization requires classifier") instead.

Reply:
No, that doesn't worth a 1201st ticket.

Reply:
Why are you adding non-actionable TODOs then?

Reply:
They are actionable when whatever person will come and start working/improving this code. It may never happen (then we save time by NOT creating an issue for it), or it may happen (then a person working on that code may benefit from this piece of information).